### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by MiaoJieF** [haofeixu/gmflow#27]
Notice that the training code will judge whether the loss function is nan, what is the reason for this and what is the effect?

**Comment by haofeixu** [comment]
The only reason is that FlyingThings3D gt flow might contain nan and thus we simply skip such samples.

**Comment by MiaoJieF** [comment]
How does this affect performance?

**Comment by haofeixu** [comment]
We can't train on nan data so skiping them is natural. I don't think this will affect performance.

**Comment by MiaoJieF** [comment]
Thank you. The reason why I have this doubt is because the previous methods do not seem to have this operation.
